packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx

```diff
@@ -16,9 +17,9 @@ const CapturePhoto: React.FC<CapturePhotoProps> = ({ initialState, onCapturePhot
 
   const showCam = useCallback(() => {
     const close = showModal('capture-photo-modal', {
-      saveFile(dataUri: string) {
-        setDataUri(dataUri);
-        onCapturePhoto(dataUri, toOmrsIsoString(new Date()));
+      saveFile(file: UploadedFile) {
+        setDataUri(file.base64Content);
+        onCapturePhoto(file.base64Content, toOmrsIsoString(new Date()));
         close();
```

Comment:
saveFile(file: UploadedFile) {
        setDataUri(file.base64Content);
        onCapturePhoto(file.base64Content, toOmrsIsoString(new Date()));
        close();
        return Promise.resolve();

Reply:
This will fix @jwnasambu